apps/landingPage/src/components/Headline.js

```diff
@@ -13,87 +13,137 @@ import airport from '../images/airport.jpg';
-              Tequila mixed with something else. Just grab a Margarita.
+type State = {|
+  +width: number,
+|};
+
+export default class Headline extends React.Component<{||}, State> {
+  state = {
+    width: 0,
+  };
+
+  componentDidMount() {
+    this.handleWindowSizeChange();
+    window.addEventListener('resize', this.handleWindowSizeChange);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleWindowSizeChange);
+  }
+
+  handleWindowSizeChange = () => {
+    const width = typeof window !== 'undefined' ? window.innerWidth : 0;
+    this.setState({ width });
+  };
+
+  render() {
+    const alignment = this.state.width > BREAKPOINTS.TABLET ? 'left' : 'center';
+    const isTablet = this.state.width <= BREAKPOINTS.TABLET;
+    const buttonSize = isTablet ? 'large' : 'normal';
+    return (
+      <HeadlineContainer>
+        <HeadlineTextContainer>
+          <Text align={alignment}>
+            <Heading type="display">
+              Ever dreamt of building the next Kiwi.com?
             </Heading>
-          </HeaderAndDesignWrapper>
-          <ContainerIcons>
-            <img src={kiwiLogo} height={43} alt="kiwi_logo" />
-            <img src={reactLogo} height={45} alt="react_logo" />
-            <img src={tequilaLogo} height={43} alt="tequila_logo" />
-            <img src={graphqlLogo} height={43} alt="graphql_logo" />
-          </ContainerIcons>
-        </ContainerTop>
-        <ContainerBottom>
-          <ContainerLeftText>
-            <Heading>Get started</Heading>
-            <WrapperText>
-              <Text size="large">
-                PLACEHOLDER!!!!!You've had a look at Tequila, but you've
-                preffered yourTequila Tequila mixed with something else. Just
-                grab Margarita.You've had a look at Tequila, but you've always
-                preffered your Tequila mixed with something else.
+          </Text>
+        </HeadlineTextContainer>
+        <Container>
+          <ContainerTop>
+            <HeaderAndDesignWrapper>
+              <BackgroundGradientAccent>
+                <GreenRectangle />
+              </BackgroundGradientAccent>
+              <Text align={alignment}>
+                <Heading type="title2">
+                  You've had a look at Tequila, but you've always preffered your
+                  Tequila mixed with something else. Just grab a Margarita.
+                </Heading>
               </Text>
-            </WrapperText>
-            <WrapperButtons>
-              <Button>
-                <LinkNoStyle
-                  href={margarita}
-                  color={defaultTokens.paletteWhite}
-                >
-                  Get started
-                </LinkNoStyle>
-              </Button>
-              <WrapperRightButton>
-                <Button type="white">
+            </HeaderAndDesignWrapper>
+            <ContainerIcons>
+              <img src={kiwiLogo} height={43} alt="kiwi_logo" />
```

Comment:
`alt` should be without `_`. It should be human readable (it can be automatically read by some scanner).

Reply:
Agree, will change, and I dont know why I am putting snake case everywhere 😓